app/reduxContent/invoke/thunks.js

```diff
@@ -95,6 +95,9 @@ export function invokeAddress(
       }
 
       const clearedOperationId = clearOperationId(res.operationGroupID);
+      const consumedGas = operationResult.consumed_gas
+        ? parseInt(operationResult.consumed_gas, 10)
```

Comment:
"10" can be assumed, no?

Reply:
![parse](https://user-images.githubusercontent.com/27675170/55983813-fe57d000-5c69-11e9-972c-31aa56d24136.png)

Reply:
I know what it is, wouldn't parseInt(n) for normal user entry be equivalent to parseInt(n, 10); I doubt people will be typing 0x7A, etc.